### PR TITLE
Update to new file browser interface

### PR DESCRIPTION
**Pull request by blink1073**
Adds tracking of most recently focused document.

### CODE REVIEW

File: src/fileopener/plugin.ts [hunk 1]
```diff
@@ -44,6 +47,13 @@ import {
 
 
 /**
+ * The class name added to focues widgets.
```
Comment (dwillmer) [inline src/fileopener/plugin.ts]: typo

**Comment by blink1073** [comment]
Fixed.

**Comment by dwillmer** [comment]
:+1:

**Comment by jasongrout** [comment]
I think that eventually, we should rename the file opener plugin to something like DocumentManager.  Tracking the currently active document is starting to be more of a document manager than just a file opener.

**Comment by blink1073** [comment]
Sure, PR coming.

**Pull request merged**